### PR TITLE
ENG-191 improve login error message

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/state/login-form/actions.js [hunk 1]
```diff
@@ -34,8 +34,9 @@ export const performLogin = (username, password) => dispatch => (
           dispatch(setLoginErrorMessage({ id: 'fcc.login.errorMessage', defaultMessage: ERROR_LOGIN_MESSAGE }));
           resolve();
         }
-      }).catch(() => {
-        dispatch(setLoginErrorMessage({ id: 'fcc.login.errorMessage', defaultMessage: ERROR_LOGIN_MESSAGE }));
+      }).catch((err) => {
+        const errorId = err.message === 'app.serverError' ? 'app.coreUnreachable' : 'fcc.login.errorMessage';
+        dispatch(setLoginErrorMessage({ id: errorId, defaultMessage: ERROR_LOGIN_MESSAGE, domain: '' }));
```
Comment (gidesan) [inline src/state/login-form/actions.js]: why adding an empty string for domain?

File: src/locales/en.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export default {
     'app.permissionDenied': 'your session expired',
     'app.usingMocks': 'App Builder is currently using mocks',
     'app.welcome': 'Welcome to Entando, {name}!',
+    'app.coreUnreachable': 'Entando core is unreachable',
```
Comment (gidesan) [inline src/locales/en.js]: Please use a more user-oriented message :)

Comment (ichalagashvili) [reply src/locales/en.js]: Can you please suggest one? :)

Comment (gidesan) [reply src/locales/en.js]: app.serverError was already fine

**Comment by entando-service** [comment]
All is well

File: src/locales/en.js (hunk 2 above)
Comment (gidesan) [reply src/locales/en.js]: any contraindication with `app.serverError`?

File: src/state/login-form/actions.js [hunk 3]
```diff
@@ -34,8 +34,14 @@ export const performLogin = (username, password) => dispatch => (
           dispatch(setLoginErrorMessage({ id: 'fcc.login.errorMessage', defaultMessage: ERROR_LOGIN_MESSAGE }));
           resolve();
         }
-      }).catch(() => {
-        dispatch(setLoginErrorMessage({ id: 'fcc.login.errorMessage', defaultMessage: ERROR_LOGIN_MESSAGE }));
+      }).catch((err) => {
```
Comment (gidesan) [inline src/state/login-form/actions.js]: What about having an error handling function that chooses the proper message to display, so the action calls it and the `LoginFormContainer` simply displays the processed message?

**Comment by entando-service** [comment]
All is well

**Pull request merged**